src/V3AstNodeDType.h

```diff
@@ -959,6 +959,30 @@ class AstMemberDType final : public AstNodeDType {
         return false;
     }
 };
+class AstNBACommitQueueDType final : public AstNodeDType {
+    // @astgen ptr := m_targetDTypep : AstNodeDType  // Type of the corresponding variable
```

Comment:
```suggestion
    // @astgen ptr := m_subDTypep : AstNodeDType  // Type of the corresponding variable
```

To match other "container" dtypes.